content/lang-support/stdlib.md

```diff
@@ -136,35 +137,36 @@ sync |  <span style="color: green">✔</span> yes  |
 sync/atomic |  <span style="color: green">✔</span> yes  | 
 syscall |  <span style="color: green">✔</span> yes  | 
 syscall/js |  <span style="color: green">✔</span> yes  | 
-testing |  [<span style="color: red">✗</span> no](#testing)  | 
-testing/iotest |  <span style="color: green">✔</span> yes  | 
+testing |  <span style="color: green">✔</span> yes  | 
+testing/iotest |  [<span style="color: red">✗</span> no](#testing-iotest)  | 
 testing/quick |  [<span style="color: red">✗</span> no](#testing-quick)  | 
 text/scanner |  <span style="color: green">✔</span> yes  | 
 text/tabwriter |  <span style="color: green">✔</span> yes  | 
 text/template |  [<span style="color: red">✗</span> no](#text-template)  | 
-text/template/parse |  [<span style="color: red">✗</span> no](#text-template-parse)  | 
+text/template/parse |  <span style="color: green">✔</span> yes  | 
 time |  <span style="color: green">✔</span> yes  | 
 unicode |  <span style="color: green">✔</span> yes  | 
 unicode/utf16 |  <span style="color: green">✔</span> yes  | 
 unicode/utf8 |  <span style="color: green">✔</span> yes  | 
 unsafe |  <span style="color: green">✔</span> yes  | 
-vendor/golang.org/x/crypto/chacha20poly1305 |  <span style="color: green">✔</span> yes  | 
```

Comment:
I would guess these errors are caused by the switch to Go modules.

    use of vendored package not allowed

I think they can be ignored, none of the existing crypto related packages have stopped compiling. And you shouldn't import these vendor directories anyway.

Reply:
I added some code to ignore vendored directories, and regenerated the imports report.

Is this good now, @aykevl ?